src/i18n.js

```diff
@@ -4,22 +4,22 @@ import { createI18n } from 'vue-i18n';
 
 // TODO we should lazyload non global translations on a per page basis
 // https://vue-i18n.intlify.dev/guide/advanced/lazy.html
-import en from '@/translations/en.json';
-// import es from '@/translations/es.json';
-// import fr from '@/translations/fr.json';
+import English from '@/translations/en.json';
+import Español from '@/translations/es.json';
+import Française from '@/translations/fr.json';
```

Comment:
Don’t use accents in the code, stick to the locale abbreviations that I had before.

Reply:
It seems like it would be nice if the languages had a display name in addition to the variable name that's used by the code. It seems like that's sortof baked into the syntax of the Vue i18n implementation, but it's strange to me that the options we display in the menu are seemingly limited to text that makes for reasonable variable names

Reply:
So how about i go with this Approch 
```
          <v-list-item
            v-for="(lang, i) in $i18n.availableLocales"
            :key="`${i}-${lang}`"
            :value="lang"
            @click="$i18n.locale = lang"
            :data-lang="lang"
          >
  -->        {{langValue[lang]}}
          </v-list-item>
```
```
const langValue = {en:'English', es:'Español'} 
```

Reply:
It’s a little trickier than that, the names in the drop-down menu would need to change by locale also so we’ll probably need a language specific locale map. Could maybe en put those as localized strings in the translation json files and use slugs.

Reply:
To simplify, let's keep the original names for now and revisit them later